clusterloader2/testing/load/daemonset.yaml

```diff
@@ -28,3 +28,6 @@ spec:
             memory: "10M"
       priorityClassName: daemonset-priorityclass-0 # Name is autogenerated, hence the -0 prefix.
       terminationGracePeriodSeconds: 1
+{{if Chance (DefaultParam .CL2_LOAD_DAEMON_SETS_DISABLE_AUTOMOUNT_DEFAULT_SERVICE_ACCOUNT_CHANCE 0)}}
```

Comment:
It's hard to imagine longer name - can we make it shorter?
- remove load (we will be merging tests sooner or later)
- remove automount (it seems quite obvious from the context)
- remove default probably too

so
CL2_DAEMON_SETS_DISABLE_SERVICE_ACCOUNT_CHANCE

(still long, but not that long)

Reply:
BTW - do we really need this param for everything? Maybe having it from just deployments is enough, since these are 90%+ of all pods?

Reply:
Done, fixed too long names.

Reply:
There are two use cases that we are enable for testing:

1. Limited number of default SA to non-zero value (here chance is required and probably we could only depend only on deployments)
1. Disable all default SA - here we need a knob to tune other kinds

For use case 2. we could either provide additional boolean value that will disable all SA, provide per kind boolean value, or provide per kind chance value.

The middle ground between 1. and 2. is having per kind chance value, as done in this PR and this way we can keep consistency. It's also the most flexible way.